src/DotVVM.Framework/ResourceManagement/IDeferableResource.cs

```diff
@@ -1,6 +1,6 @@
 namespace DotVVM.Framework.ResourceManagement
 {
-    public interface IDeferableResource
+    public interface IDeferrableResource
```

Comment:
I am not sure whether this could break someone. RFC @exyi @tomasherceg

Reply:
This is breaking change. So we can merge it only into version 4.0.